src/app/metadata/services/well-known.service.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Well-Known Value Service
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+import { httpResource } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { ConfigService } from '@app/shared/services/config.service';
+import {
+  emptyStorageAliasDecodes,
+  storageAliasDecodes,
+} from '../models/storage-aliases';
+
+/**
+ * Well-Known Value query service
+ *
+ * This service provides the functionality to fetch well-known values.
+ */
+@Injectable({ providedIn: 'root' })
+export class WellKnownValueService {
+  #config = inject(ConfigService);
+  #wkvsUrl = this.#config.wkvsUrl;
+
+  #storageAliasDecodesUrl = `${this.#wkvsUrl}/storage_alias_decodes`;
```

Comment:
I think the path is wkvsUrl/values/storage_labels (note the /values/ part).

Reply:
No, because WKVS Url already contains /values/

Reply:
Makes sense, but for consistency I think it's better to change this. All our other URLs in the settings refer to the base addresses of the respective service, even if the service has only one resource/endpoint. And in archive test bed the `wkvs_url` config setting is also the base address, `/.well-known`.

In the service itself, you can add an internal property `#values_url` that would be `#wkvs_url` + `/values`. Similarly to how we do it with `#workPackagesUrl` and `#usersUrl` in the work-package.service.